apps/composer-ui/package-scripts.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  scripts: {
+    default: 'next dev',
```

Comment:
We will sync these similar to the commands of the backend.

Reply:
ok